build.yaml

```diff
@@ -3732,13 +3794,39 @@ steps:
       time retry ./gradlew --version
       make wheel upload-artifacts DEPLOY_REMOTE=origin
 
-      bash scripts/deploy.sh $(cat /io/hail_pip_version) \
+      bash scripts/deploy.sh $HAIL_PIP_VERSION \
                              $(cat /io/hail_version) \
                              $(cat /io/git_version) \
                              origin \
                              /io/repo/hail/build/deploy/dist/hail-*-py3-none-any.whl \
-                             /io/github-oauth \
-                             hailgenetics_hail_image
+                             /io/github-oauth
+
+      retry() {
+          "$@" ||
+              (sleep 2 && "$@") ||
+              (sleep 5 && "$@");
+      }
+
+      retry skopeo copy {{ hailgenetics_hail_image.image }} docker.io/hailgenetics/hail:$HAIL_PIP_VERSION
+      retry skopeo copy {{ hailgenetics_hail_image.image }} gcr.io/hail-vdc/hailgenetics/hail:$HAIL_PIP_VERSION
```

Comment:
The `gcr.io/hail-vdc` should be `{{ global.docker_prefix }}`

Reply:
ah yes.

Reply:
OK, so here's my plan:

I'll make a separate step which copies into `{{ global.docker_prefix }}` (my ultimate goal is for python-dill to be tagged with hail version and include hail). I'll keep the docker.io/hailgenetics/hail stuff in deploy.

Reply:
Hmm. this ain't right.